server/public/styles.css

```diff
@@ -14,11 +14,25 @@ header {
   color: #985FBA;
 }
 
-.card img {
+.card {
+  cursor: pointer;
+}
+
+.card img, .details img {
   object-fit: contain;
 }
 
 .card .image {
   height: 250px;
   display: flex;
 }
```

Comment:
You shouldn't add styling to css library classes (bootstrap). You should make a custom class name to add these additional styles to those. You don't want to change how the core library functions it will cause issues with other developers and even yourself down the road.